app/src/main/java/aloksharma/ufl/edu/stash/DisplayMessageActivity.java

```diff
@@ -0,0 +1,58 @@
+package aloksharma.ufl.edu.stash;
+
+/**
+ * Created by amazon on 10/7/15.
+ */
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+
+import android.view.MenuItem;
+import android.widget.TextView;
+
+
+
+public class DisplayMessageActivity extends Activity {
```

Comment:
This would be  a activity used for onclick button activity followed from addstash. It might be replaced later, with Stash view . This is currently a helper if needed for addstash. 

Updated the name temporarily it as AddStashHelper.